m4_sessions/1803-inning/group_3.md

```diff
@@ -1,13 +1,23 @@
 Group 3
 
-Group Members:
+Group Members: Alex Bonder, Jeff Goodall
 
-Name of Session:
+Name of Session: How to be an advocate for those that don't have a voice.
 
 Session Description (Why should students come to this session):
-
+  Advocacy is a driving force in promoting diversity within the workplace.
+  Some people aren't comfortable asking questions even though the environment emphasises that all voices are welcome. How do we overcome this?
+  
+  
 Session Outcomes (What will students learn/leave with?):
+  Leave with the understanding of how someone without a voice might feel day to day. and Some ideas on how to actively participate in being an advocate for those people.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+  15 minutes - Introductions
+  15 minutes - Ask questions and see the demographics of who volunteers to answer.
+  15 minutes - Ted Talk/Video
+  30 minutes - Open discussion - Who are the people who don't have a voice - write on board
```

Comment:
I'm not 100% sure about this...I think you might just add a lot of types of people on the board.  I might instead do a specific discussion around what lenses people feel aware of and ready to advocate for others.  Maybe also identify the lenses that people aren't aware of and discuss how we could be better allies.

You might start with having people reflect on when someone has been an advocate for them, what it felt like not to have a voice, and what it felt like when someone was able to be a voice for them